src/profiles.jl

```diff
@@ -79,6 +83,22 @@ function Base.getindex(op::OperationalProfile, i::TimeStructurePeriod)
     return error("Type $(typeof(i)) can not be used as index for an operational profile")
 end
 
+function _print_values(vals; delim = ',', max_elems = 10)
+    n = length(vals)
+    sep = delim * " "
+    if n <= max_elems
+        return join(vals, sep)
+    end
+    nshow = round(Int, max_elems / 2)
+    seq1 = vals[1:nshow]
+    seq2 = vals[(end-nshow+1):end]
+    return join(seq1, sep) * sep * "... " * sep * join(seq2, sep)
```

Comment:
```suggestion
    return join(seq1, sep) * sep * "..." * sep * join(seq2, sep)
```

I think it looks better without spaces before `...`.

Reply:
You removed the comma in general? It is more in line with how Julia represents the values, so I think it is ok.